src/app/components/root-injectable/root-injectable.component.html

```diff
@@ -0,0 +1,5 @@
+<div>
+  <h3>Root Injectable</h3>
+  <h5>This component instance</h5>
+  <p>{{this.rootService.persistentData}}</p>
```

Comment:
No es buena práctica usar "this" en el HTML. 
Tampoco acceder directamente a servicios, favor de corregir en todas partes.